openlibrary/plugins/admin/code.py

```diff
@@ -111,7 +112,7 @@ def reload(self, servers):
             s = web.rstrips(s, "/") + "/_reload"
             yield "<h3>" + s + "</h3>"
             try:
-                response = urllib.request.urlopen(s).read()
+                response = requests.get(s).content
                 yield "<p><pre>" + response[:100] + "</pre></p>"
             except:
```

Comment:
These three lines present an interesting one... .content would return bytes.
Adding str and bytes is OK in Py2 but raises TypeError Py3
The baRe exception (bad idea) sweeps the TypeError under the carpet so Python 3 always yields the trace back.

We want .text, not .context in this instance.